### PR TITLE
Update guru-tutorial.md

### PR DESCRIPTION
**Pull request by duffyguru**
added three hyperlinks to useful Guru URLs.

### CODE REVIEW

File: docs/identity/saas-apps/guru-tutorial.md [hunk 1]
```diff
@@ -165,4 +165,6 @@ In this section, you test your Microsoft Entra single sign-on configuration with
 
 ## Next Steps
 
-Once you configure Guru you can enforce session control, which protects exfiltration and infiltration of your organization's sensitive data in real time. Session control extends from Conditional Access. [Learn how to enforce session control with Microsoft Defender for Cloud Apps](/cloud-app-security/proxy-deployment-any-app).
\ No newline at end of file
+Once you configure Guru you can enforce session control, which protects exfiltration and infiltration of your organization's sensitive data in real time. Session control extends from Conditional Access. [Learn how to enforce session control with Microsoft Defender for Cloud Apps](/cloud-app-security/proxy-deployment-any-app).
+
+For any additional information, please visit [Guru's Help Center](https://help.getguru.com/s/)
```
Comment (duffyguru) [inline docs/identity/saas-apps/guru-tutorial.md:170]: I forgot to add a period to the end of this sentence - are you able to add one please?

**Comment by duffyguru** [comment]
hey @Jak-MS 

are you able to have someone review this?

**Comment by Jak-MS** [comment]
@jeevansd - nudge. can you review please?

**Comment by v-hgampala** [comment]
@duffyguru, thanks for raising the request!!
we have created a separate pull request for your changes in our private repo for better content verification and validation, the changes will live within 2-3 working days.
#please-close

**Pull request closed without merging**